h/api/views.py

```diff
@@ -76,9 +76,18 @@ def index(context, request):
 @api_config(context=Root, name='search')
 def search(request):
     """Search the database for annotations matching with the given query."""
+    search_normalised_uris = request.feature('search_normalised')
```

Comment:
Can this really usefully be feature flagged anyway? I mean, I can see how it can be turned on or off for a deployment, but can it really be on for some people but not for others? The way it's described is that this argument says that we can assume all the URIs have a normalized copy stored and indexed, but that's not true if it only happens for some users' annotations.